extensions/commands/art/cmd_build_info.py

```diff
@@ -352,14 +352,19 @@ def _add_default_arguments(subparser, is_bi_create=False, is_bi_create_bundle=Fa
     return subparser
 
 
+def text_formatter(info):
+    if info:
+        cli_out_write(info)
+
+
 @conan_command(group="Artifactory")
 def build_info(conan_api: ConanAPI, parser, *args):
     """
     Manages JFrog Build Info (https://www.buildinfo.org/)
     """
 
 
-@conan_subcommand()
+@conan_subcommand(formatters={"text": text_formatter})
```

Comment:
Being the build-info a json file this is a bit confusing, but I understand this is the default formatter, so 👍

Reply:
yes, I agree, It's a bit confusing